.github/workflows/ci.yaml

```diff
@@ -75,22 +75,16 @@ jobs:
           # As of June 2020, confirmed errors still result
           # - go-version: 1.13.x
           #   dirs: _integrations/nrmicro
-          - go-version: 1.13.x
-            dirs: _integrations/nrnats
-          - go-version: 1.13.x
-            dirs: _integrations/nrstan
+          # As of Jul 2022, we have depreciated the legacy nrnats,nrmssql, nrzap, and nrstan integrations tests.
+          # These tests still exist under the v3 versions of the integrations.
           - go-version: 1.13.x
             dirs: _integrations/logcontext
-          - go-version: 1.13.x
-            dirs: _integrations/nrzap
           - go-version: 1.13.x
             dirs: _integrations/nrhttprouter
           - go-version: 1.13.x
             dirs: _integrations/nrb3
           - go-version: 1.13.x
             dirs: _integrations/nrmongo
-          - go-version: 1.13.x
-            dirs: _integrations/nrmssql
 
           # v3 agent
           - go-version: 1.7.x
```

Comment:
Lets remove tests for the v3 agent for go version 1.7 and 1.12 as well. They are well outside of the deprecation window.

Reply:
Should we remove just the 1.7 and 1.12 v3 tests or 1.7 through 1.12?

Reply:
You know what, let's handle this in a separate issue. I don't want to get stuck log rolling this. I also want to make test for 1.16, 1.17, 1.18. We should also separate the log context tests out into their own test. Idk why thats bundled.